### PR TITLE
[DOCA] Run tests in containers on ubuntu22.04

### PR DESCRIPTION
**Pull request by dpressle**
## Description
Run tests in containers

##### What
Convert test processes to run on k8s containers instead of static bare metal server
and run on ub22.04 instead of old RHEL OS

##### Why ?
For doca integration

##### How ?
_It is optional but for complex PRs please provide information about the design,
architecture, approach, etc._

## Change type
What kind of change does this PR introduce?
- [ ] Bugfix
- [ ] Feature
- [ ] Code style update
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [x] CI related changes
- [ ] Documentation content changes
- [ ] Tests
- [ ] Other

## Check list
- [ ] Code follows the style de facto guidelines of this project
- [ ] Comments have been inserted in hard to understand places
- [ ] Documentation has been updated (if necessary)
- [ ] Test has been added (if possible)

### CODE REVIEW

**Comment by vialogi** [comment]
bot:retest

**Comment by dpressle** [comment]
bot:retest

File: contrib/jenkins_tests/test.sh [hunk 1]
```diff
@@ -93,15 +88,16 @@ if [ ! -z "${test_remote_ip}" ] ; then
...
+    # Get IPv4/6 addresses for net1 interface which is a VF of MLNX NIC
+    test_ip_list="eth_ip4:$(ip -f inet addr show net1 | awk '/inet / {print $2}' | cut -d/ -f1)"
+    # test_ip_list="${test_ip_list} eth_ip6:$(ip -f inet6 addr show net1 | awk '/inet6 / {print $2}' | cut -d/ -f1)"
+    if [ -z "$test_ip_list" ]; then
+        echo "ERROR: Cannot get IPv4/6 address of net1 interface!"
+        exit 1
+    fi
+
+    # start the ssh server as verifyer.pl requiers
+    /etc/init.d/ssh start
```
Comment (igor-ivanov) [inline contrib/jenkins_tests/test.sh:100]: please fix formatting

File: contrib/jenkins_tests/gtest.sh [hunk 2]
```diff
@@ -25,6 +42,13 @@ if [[ -z "${MANUAL_RUN}" ]]; then
 	gtest_app="$PWD/tests/gtest/gtest"
 	gtest_lib=$install_dir/lib/${prj_lib}
 	opt2=''
+
+    gtest_ip_list_1=$(ip -f inet addr show net1 | awk '/inet / {print $2}' | cut -d/ -f1)
+    gtest_ip_list_2=$(ip -f inet addr show net2 | awk '/inet / {print $2}' | cut -d/ -f1)
+    gtest_opt="--addr=${gtest_ip_list_1},${gtest_ip_list_2}"
+    # gtest_ipv6_list_1=$(ip -f inet6 addr show net1 | awk '/inet6 / {print $2}' | cut -d/ -f1)
+    # gtest_ipv6_list_2=$(ip -f inet6 addr show net2 | awk '/inet6 / {print $2}' | cut -d/ -f1)
+    # gtest_opt_ipv6="--addr=${gtest_ipv6_list_1},${gtest_ipv6_list_2} -r fdff:ffff:ffff:ffff:ffff:ffff:ffff:ffff" # Remote - Dummy Address
```
Comment (igor-ivanov) [inline contrib/jenkins_tests/gtest.sh:51]: please fix formatting

File: contrib/jenkins_tests/vg.sh [hunk 3]
```diff
@@ -21,15 +21,26 @@ rc=$?
...
+    if [ ! -z "$(do_get_ip 'eth')" ]; then
+    	test_ip_list="${test_ip_list} eth_ip6:$(do_get_ip 'inet6')"
+    fi
+else
+    test_ip_list="eth_ip4:$(ip -f inet addr show net1 | awk '/inet / {print $2}' | cut -d/ -f1)"
+    # test_ip_list="${test_ip_list} eth_ip6:$(ip -f inet6 addr show net1 | awk '/inet6 / {print $2}' | cut -d/ -f1)"
+    if [ -z "$test_ip_list" ]; then
+        echo "ERROR: Cannot get IPv4 address of net1 interface!"
+        exit 1
+    fi
```
Comment (igor-ivanov) [inline contrib/jenkins_tests/vg.sh:41]: please following existing `tab` formatting

File: contrib/jenkins_tests/test.sh (hunk 1 above)
Comment (dpressle) [reply contrib/jenkins_tests/test.sh]: done

File: contrib/jenkins_tests/vg.sh (hunk 3 above)
Comment (dpressle) [reply contrib/jenkins_tests/vg.sh]: done

File: contrib/jenkins_tests/gtest.sh (hunk 2 above)
Comment (dpressle) [reply contrib/jenkins_tests/gtest.sh]: done

**Pull request closed without merging**